Terminal.Gui/Windows/Wizard.cs

```diff
@@ -103,6 +103,14 @@ public WizardStep (ustring title)
 					helpTextView.SetNeedsDisplay ();
 				};
 
+				scrollBar.OtherScrollBarView.ChangedPosition += () => {
```

Comment:
Please don't mix other fixes with PRs like this.

Reply:
Some of the issues were interconnected and when creating a new branch I based on another already created and submitted. Sorry.